### PR TITLE
Improve Email Field Requirement for Inviting Friends and Link Bell Icon to Newsletter Page

### PR DESCRIPTION
**Pull request by willy1989cv**
#190 

* [x] make the email field required for me when click to enter a message in the message box.
* [x] If I do not click to enter a message, the email field should remain optional. 
* [x] I am redirected to the newsletter subscription page
extra:
* [x] fix build when updating tdc member.

### CODE REVIEW

File: frontend/src/components/onboarding-steps/InviteUsersStep.tsx [hunk 1]
```diff
@@ -20,17 +31,19 @@ export default ({
...
           <MultipleSelector
             removeSuggestions
             hidePlaceholderWhenSelected
             transformInputValueInLowercase
-            onChange={(x) =>
-              form.setValue(
+            onChange={(x) => {
+              return form.setValue(
                 "newUsersEmailsToInvite",
-                Array.from(new Set(x.map((x) => x.value)))
```
Comment (Gutts-n) [inline frontend/src/components/onboarding-steps/InviteUsersStep.tsx:31]: Why did you remove the set of this line? This set was there to prevent repeated emails from being sent on the field.

**Review by Gutts-n — CHANGES_REQUESTED** [review]

File: frontend/src/components/onboarding-steps/InviteUsersStep.tsx (hunk 1 above)
Comment (vvilly89) [reply frontend/src/components/onboarding-steps/InviteUsersStep.tsx:31]: fixed @Gutts-n

Comment (willy1989cv) [reply frontend/src/components/onboarding-steps/InviteUsersStep.tsx:31]: @Gutts-n fixed

**Review by Gutts-n — APPROVED** [review]
LGTM
